app/controllers/api/v2/hosts_controller.rb

```diff
@@ -43,8 +43,8 @@ class HostsController < V2::BaseController
 
       def index
         @hosts = action_scope_for(:index, resource_scope_for_index)
-
         if params[:thin]
+          @subtotal = @hosts.count
```

Comment:
maybe i'm missing it, but where does pagination get sets? or in other words, are you sure that @hosts is pre - pagination (so count actually counts all of the hosts?) looking at this code, it actually looks you can move this line even after the ````@hosts = @hosts.reorder...``` and simply count in memory vs an SQL query.

Reply:
@hosts is post pagination, and uses `will-paginate` count [implementation](https://github.com/mislav/will_paginate/blob/5d92ee219d4a941a18c0de8a24a11973548495ac/lib/will_paginate/active_record.rb#L81)
 however, looks like it better to use `total_entries`, it doesn't make an additional query.